### PR TITLE
Modifying all things associated with settings page

### PR DESCRIPTION
**Pull request by itsAndL**

### CODE REVIEW

**Comment by itsAndL** [comment]
Screenshot 2023-06-19 130859

**Comment by itsAndL** [comment]
@mohamedrez. if you notice at this point i'm not using the settings page, but instead just showing the pages for (roles, sites, holidays)

File: app/models/role.rb [hunk 1]
```diff
@@ -3,5 +3,5 @@ class Role < ApplicationRecord
 
   validates :name, presence: true
 
-  enum name: {hr: 1, admin: 2, accountant: 3, manager: 4}
+  enum name: {admin: 0, hr: 1, accountant: 2, manager: 3}
```
Comment (mohamedrez) [inline app/models/role.rb:6]: Why is this change needed ?

**Review by mohamedrez — CHANGES_REQUESTED** [review]

File: app/models/role.rb (hunk 1 above)
Comment (itsAndL) [reply app/models/role.rb:6]: i just wanna put admin in the first of select menu

File: app/policies/admin_hr_policy.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
+class AdminHrPolicy < ApplicationPolicy
```
Comment (mohamedrez) [inline app/policies/admin_hr_policy.rb:1]: Can you avoid inhertiance and instead use composition?

**Review by mohamedrez — CHANGES_REQUESTED** [review]

**Review by mohamedrez — APPROVED** [review]

**Pull request merged**